### PR TITLE
Setup React Enviroment for Math Magicians project

### PR DESCRIPTION
**Pull request by rominana**
In this project I: :smile: 
- Updated Node.js for Linux distributions.
- Installed React and set up the environment.
- Installed React linters.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  }
+}
```
Comment (DeliceLydia) [inline package.json]: - [x] Please add react dependencies you can create your react app using `npx create-react-app my-app`

- [x] Please add scripts to be able to run the app they are actually added when creating the app
```javascript
"scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
```

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @rominana, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: package.json (hunk 1 above)
Comment (rominana) [reply package.json]: All changes have been applied :smile:
Thanks for the feedback

**Comment by rominana** [comment]
All the suggested changes on the package.json file configuration have been applied :heavy_check_mark:

**Review by mikethreels — APPROVED** [review]
Hi Romana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**